src/Symfony/Component/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -132,8 +156,10 @@ protected function describeApplication(Application $application, array $options
             }
 
             $this->write("\n\n");
-            $this->write(implode("\n", array_map(function ($commandName) {
-                return '* `'.$commandName.'`';
+            $this->write(implode("\n", array_map(function ($commandName) use ($description) {
+                $anchor = '#'.strtr($description->getCommand($commandName)->getName(), array(':' => ''));
```

Comment:
Maybe do simple `str_replace` here. And this can actually be `$commandName` right? :confused: